domain/src/main/java/org/oppia/android/domain/platformparameter/PlatformParameterModule.kt

```diff
@@ -171,4 +173,12 @@ class PlatformParameterModule {
       ENABLE_EXTRA_TOPIC_TABS_UI_DEFAULT_VALUE
     )
   }
+
+  @Provides
+  @EnableConfigurationChange
+  fun provideEnableConfigurationChange(): PlatformParameterValue<Boolean> {
```

Comment:
This will also need to be added to https://github.com/oppia/oppia-android/blob/develop/domain/src/main/java/org/oppia/android/domain/platformparameter/PlatformParameterAlphaKenyaModule.kt.

Reply:
Done

Reply:
This doesn't seem complete--I don't see any changes in that module file. Am I missing something?

Reply:
I have already added in [PlatformParameterAlphaKenyaModule.kt](https://github.com/oppia/oppia-android/blob/2f45431aa16436928bbe633c8b9a8c4bc9b0146c/domain/src/main/java/org/oppia/android/domain/platformparameter/PlatformParameterAlphaKenyaModule.kt#L172) can you please check it again. Thanks

Reply:
Hmm, I think I missed it before but I do see it now. Thanks!